README.md

```diff
@@ -10,6 +10,7 @@ This repository provides a collection of examples that demonstrate how you can m
 
 | Example | Description |
 | ------- | ----------- |
-| [Camel Quarkus](./camel-quarkus/) | A simple Camel Quarkus application example. |
+| [Camel Quarkus for Kubernetes](./camel-quarkus-kubernetes/) | A simple Camel Quarkus application example deployable on all variants of Kubernetes. |
+| [Camel Quarkus for OpenShift](./camel-quarkus-openshift/) | A simple Camel Quarkus application example specifically targeted for deployment on OpenShift. |
```

Comment:
As far as I understand the addition of camel-quarkus-openshift is only for the quarkus-openshift plugin to automatically configure the deployment to be discovered by Hawtio right?